admin_guide/scheduling/taints_tolerations.adoc

```diff
@@ -315,7 +315,7 @@ To maintain the existing link:https://kubernetes.io/docs/admin/node/#node-contro
 [[admin-guide-taints-daemonsets]]
 == Daemonsets and Tolerations
 
-link:https://kubernetes.io/docs/admin/daemons/[DaemonSet] pods are created with `NoExecute` tolerations for `node.alpha.kubernetes.io/unreachable` and `node.alpha.kubernetes.io/notReady`
+link:https://kubernetes.io/docs/admin/daemons/[DaemonSet] pods are created with `NoExecute` tolerations for `node.kubernetes.io/unreachable` and `node.kubernetes.io/notReady`
```

Comment:
notready->not-ready

Reply:
@aveshagarwal Changed all instances of `node.kubernetes.io/notReady` to `node.kubernetes.io/not-Ready`. Is this correct?